java/core/src/main/java/co/worklytics/psoxy/gateway/impl/oauth/CertificateGrantTokenRequestBuilder.java

```diff
@@ -29,6 +30,19 @@ enum ConfigProperty implements ConfigService.ConfigProperty {
         PRIVATE_KEY
     }
 
+    @Override
+    public boolean useSharedToken() {
+        Optional<String> useSharedTokenConfig = config.getConfigPropertyAsOptional(RefreshTokenTokenRequestBuilder.ConfigProperty.USE_SHARED_TOKEN);
```

Comment:
Missing on previous PR to add this; otherwise access token was never written

Reply:
this could be a one-liner